### PR TITLE
{bio}[foss/2020b] scanpy v1.8.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2605.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/e8082554968abbc54931bcf51ca3359d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3507.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5745325755a3cea8f325e2eabef5a387 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.6
See https://gist.github.com/a8fe0ee344dccb5a601baa74fad886f4 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/eca8b42cf8980b92290027586d954791 for a full test report.

File: easybuild/easyconfigs/s/scanpy/scanpy-1.8.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    ('tqdm', '4.56.2'),
+    ('leidenalg', '0.8.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('natsort', '7.1.1', {
+        'checksums': ['00c603a42365830c4722a2eb7663a25919551217ec09a243d3399fa8dd4ac403'],
+    }),
```
Comment (verdurin) [inline easybuild/easyconfigs/s/scanpy/scanpy-1.8.1-foss-2020b.eb:35]: Does the comment from the `2021a` version about `xlrd` not apply here too?

Comment (SebastianAchilles) [reply easybuild/easyconfigs/s/scanpy/scanpy-1.8.1-foss-2020b.eb:35]: `Python/3.8.6` uses `xlrd v1.2.0`:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.8.6-GCCcore-10.2.0.eb#L190

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfl01.jusuf - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/51e054d4ce38477f26d256abd190a221 for a full test report.

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**